docs/assets/send-message.txt

```diff
@@ -0,0 +1,15 @@
+SAS -> "API gateway": send message request
+"API gateway" -> "API Functions": forward message
```

Comment:
you should also add activation/deactivation instructions (as in the previous diagram), e.g.:

```puml
SAS -> "API gateway": send message request
activate SAS

"API gateway" -> "API Functions": forward message
"API Functions" -> "Database": save message
"Database" -> "API Functions": message saved
"API Functions" -> "API gateway": return message

"API gateway" -> "SAS": return queued message id
deactivate SAS

"API Functions" -> "Notification queue": map message to notification
"Notification queue" -> "API Functions": trigger notification function
"API Functions" -> "Channel queue": dispatch notification to channel
"Channel queue" -> "API Functions": trigger channel function

"API Functions" -> "Channel": send notification to channel
"Channel" -> "API Functions": return notification status
"API Functions" -> "Database": save notification status
```